docs/workflows/input_output.md

```diff
@@ -2,26 +2,25 @@
+The Cumulus Message Adapter and Cumulus Message Adapter libraries help task developers integrate their tasks into a Cumulus workflow. These libraries adapt input and outputs from tasks into the Cumulus Message format. The Scheduler service creates the initial event message by combining the collection configuration, external resource configuration, workflow configuration, and deployment environment settings.  The subsequent workflow messages between tasks must conform to the message schema. By using the Cumulus Message Adapter, individual task Lambda functions only receive the input and output specifically configured for the task, and not non-task-related message fields.
 
-The Cumulus Message Adapter libraries are called by the tasks with a callback function containing the business logic of the task as a parameter. They first adapt the incoming message to a format more easily consumable by Cumulus tasks, then invoke the task, and then adapt the task response back to the Cumulus message protocol to be sent to the next task.  
+The Cumulus Message Adapter libraries are called by the tasks with a callback function containing the business logic of the task as a parameter. They first adapt the incoming message to a format more easily consumable by Cumulus tasks, then invoke the task, and then adapt the task response back to the Cumulus message protocol to be sent to the next task.
 
-A task's Lambda function can include a Cumulus Message Adapter library which constructs input/output messages and resolves task configurations. In the Lambda function configuration file lambdas.yml, a task Lambda function can be configured to use Cumulus Message Adapter, for example:
+A task's Lambda function can be configured to include a Cumulus Message Adapter library which constructs input/output messages and resolves task configurations. In the Lambda function configuration file lambdas.yml, a task Lambda function can be configured to use Cumulus Message Adapter, for example:
 
     DiscoverPdrs:
       handler: index.handler
-      useMessageAdapter: true  
+      useMessageAdapter: true
 
-Input to task application code is a json object with keys:
-* `input`: By default, the incoming payload is the payload from previous task, or it can be a portion of the payload as specified in the task configuration.
+Input to the task application code is a json object with keys:
+* `input`: By default, the incoming payload is the payload output from the previous task, or it can be configured to be any portion of the payload using the `config` key.
```

Comment:
I'm confused about saying you can configure the output using the `config` key. I would just link to the `3. Resolve task output` section.

Reply:
So link to 3. Resolve task  and just have `By default, the incoming payload is the payload output from the previous task, or it can be configured to be any portion of the payload.`

Reply:
Oh actually re-reading it I kind of get what you're saying. My initial confusion was I read it as using the `config` key in the yml, but you mean you'd be able to access it using the `config` key in the task. The problem is you're missing a step. You can configure the output to put whatever portion of it into `meta` and then from there you can use that down the road in another step.